packages/components/src/components/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { Pagination } from "./Pagination";
+
+// eslint-disable-next-line lodash/prefer-noop, @typescript-eslint/no-empty-function
+const NOOP = (): void => {};
```

Comment:
Was there a reason you couldn't use noop from lodash?

Reply:
Nope! :) I changed it.